backend/bitboxbase/updater/updater.go

```diff
@@ -83,7 +108,150 @@ func (updater *Updater) Connect(address string, bitboxBaseID string) error {
 		return errors.New("updater: Unexpected Response Body Bytes")
 	}
 	updater.running = true
-	go listenWebsocket(updater, bitboxBaseID)
+	updater.log.Printf("connecting to base websocket")
+	client, _, err := websocket.DefaultDialer.Dial("ws://"+updater.address+"/ws", nil)
+	if err != nil {
+		updater.log.Printf("Websocket dial failed: %s", err.Error())
```

Comment:
return `errp.WithStack(err)` here.

the withstack wrapper is needed for all externally generated errors (stdlib or third party packages). Our own (`errp.New(...)`) automatically have a stack.